### PR TITLE
feat(service-version): adjust docs [tdx-2281]

### PR DESCRIPTION
**Pull request by tomaszwylezek**
Signed-off-by: Tomasz Wylężek <tomwylezek@gmail.com>

### Summary

Adjust content related to service version lifecycle.
For now the work is behind FF and when the feature is GA then we will merge that docs adjustment

### Reason
TDX-2281

### Testing

Definitely want layout / readability / if proper place opinions about what I wrote.
https://deploy-preview-4334--kongdocs.netlify.app/konnect/servicehub/service-versions/

### CODE REVIEW

**Comment by cloudjumpercat** [comment]
@tomaszwylezek Is this content ready for a review? If everything is finalized, could you change it from the draft state to ready for review? After we review it, we won't merge it until you give us the OK. Thanks!

**Comment by acgoldsmith** [comment]
update from discussion w/ Sunny: this is expected to GA before Summit.

File: app/konnect/servicehub/service-versions.md [hunk 1]
```diff
@@ -57,10 +65,26 @@ From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.co
...
 1. Click **Create** to save.
 
+
+## Managing a service version lifecycle
+
+Changing service version status will have an impact on its visibility in Dev Portal. 
+
+From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service, then selects its service version which status you would like to adjust. Then follow these steps:
+
+1. From the **Service version actions** drop-down menu, select **Edit version status**.
```
Comment (cloudjumpercat) [inline app/konnect/servicehub/service-versions.md:75]:
```suggestion
1. Click  **Service version actions** > **Edit version status**.
```

File: app/konnect/servicehub/service-versions.md [hunk 2]
```diff
@@ -57,10 +65,26 @@ From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.co
...
+
+2. In dialog you can select appropriate status. Next to each status there is a brief explanation which pops up on icon hover.
```
Comment (cloudjumpercat) [inline app/konnect/servicehub/service-versions.md:77]:
```suggestion
1. Select a service version status.
```

File: app/konnect/servicehub/service-versions.md [hunk 3]
```diff
@@ -57,10 +65,26 @@ From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.co
...
+
+    1. In some cases there might be an additional step to fulfill.
```
Comment (cloudjumpercat) [inline app/konnect/servicehub/service-versions.md:79]: @tomaszwylezek What other steps would there be here?

File: app/konnect/servicehub/service-versions.md [hunk 4]
```diff
@@ -57,10 +65,26 @@ From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.co
...
+
+3. Click **Save** to adjust status of service version.
```
Comment (cloudjumpercat) [inline app/konnect/servicehub/service-versions.md:81]:
```suggestion
1. Click **Save**.
```

File: app/konnect/servicehub/service-versions.md [hunk 5]
```diff
@@ -57,10 +65,26 @@ From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.co
...
+
 ## Delete a service version
 
 Deleting a service version permanently removes it and its implementation, routes, and plugins from the Service Hub.
 
 From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service version, then delete it:
```
Comment (cloudjumpercat) [inline app/konnect/servicehub/service-versions.md:87]:
```suggestion
1. From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select the service version you want to delete.
```

File: app/konnect/servicehub/service-versions.md [hunk 6]
```diff
@@ -57,10 +65,26 @@ From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.co
...
 
-* From the **Version actions** drop-down menu, select **Delete**, then confirm deletion in the dialog.
+* From the **Service version actions** drop-down menu, select **Delete service version**, then confirm deletion in the dialog.
```
Comment (cloudjumpercat) [inline app/konnect/servicehub/service-versions.md:89]:
```suggestion
1. Click **Service version actions** > **Delete service version**.
```

File: app/konnect/servicehub/service-versions.md [hunk 7]
```diff
@@ -57,10 +65,26 @@ From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.co
...
+* Confirm deletion by typing name of service version and click **Yes, delete**
```
Comment (cloudjumpercat) [inline app/konnect/servicehub/service-versions.md:90]:
```suggestion
1. Confirm the deletion by typing the name of the service version and clicking **Yes, delete**.
```

**Review by cloudjumpercat — CHANGES_REQUESTED** [review]
Suggested a few small changes.

File: app/konnect/servicehub/service-versions.md [hunk 8]
```diff
@@ -25,6 +25,14 @@ ready to test, you'd create v2 in `staging` before finally creating v2 in
 Instead, create a new version of the service in the new environment when you're
 ready to move to it.
 
+Every service version may be in one of the states: "Published", "Deprecated" and "Unpublished":
+* **Published**: This indicates that the service version is ready to be shared with the API consumers. It is exposed to Dev Portal where developers can request access to consume it via the {{site.base_gateway}}. This is default state.
+* **Deprecated**: This indicates that the service version will be deprecated soon. It will still be exposed to the Dev Portal and receive API request via {{site.base_gateway}}. Banner with information about deprecation will be displayed at the top of the service version page in Dev Portal.
```
[the remark this answers is not in the document]
Comment (rosskukulinski) [reply app/konnect/servicehub/service-versions.md:30]: I think we should also indicate that Dev Portal users with applications registered to this service version will receive an email notification that this version has been deprecated.

**Comment by cloudjumpercat** [comment]
@tomaszwylezek Could you review my suggestions when you have the time? Thanks!

File: app/konnect/servicehub/service-versions.md (hunk 3 above)
Comment (tomaszwylezek) [reply app/konnect/servicehub/service-versions.md:79]: In the case of changing the version status into "deprecated" - we have one more step in which the user is displayed information about that there will be - check video here - https://github.com/Kong/khcp-ui/pull/2642

**Comment by tomaszwylezek** [comment]
@cloudjumpercat hey, 
First -  I want to apologize to you that I did it just now - I must have missed that notification somehow. 😿 
Second - Introduced all of yours suggestions and answered questions

File: app/konnect/servicehub/service-versions.md [hunk 9]
```diff
@@ -57,10 +65,29 @@ From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.co
...
+
+1. From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), choose the service you want to manage the lifecycle of.
+
+2. Select the service version you want to manage.
+
+3. Click  **Service version actions** > **Edit version status**.
+
+4. Select a service version status.
+   
+    1. In some cases there might be an additional step to fulfill.
```
Comment (cloudjumpercat) [inline app/konnect/servicehub/service-versions.md:81]:
```suggestion
    1. Deprecation only: Click **Save** on the dialog to deprecate the service version and send consumers an email notification.
```

**Review by cloudjumpercat — APPROVED** [review]
LGTM!

**Comment by cloudjumpercat** [comment]
@tomaszwylezek No worries! Do you know when this will go GA? Just wondering when I should publish this. I know they said around Summit time, but I didn't know which day that would be exactly. Thanks!

**Comment by tomaszwylezek** [comment]
@cloudjumpercat 
I just talked with @nateslo about it. He will check and confirm when it should be published

**Comment by cloudjumpercat** [comment]
@tomaszwylezek @nateslo Hi! Just checking in to see if these docs can be published now that Summit is over. Thanks!

**Comment by nateslo** [comment]
> @tomaszwylezek @nateslo Hi! Just checking in to see if these docs can be published now that Summit is over. Thanks!

Hi @cloudjumpercat , we are planning on enabling the feature by the end of the week. We are doing a little more testing in the next day or so.

**Pull request merged**